lib/worker.js

```diff
@@ -2,14 +2,14 @@ const EventEmitter = require('events').EventEmitter;
 const util = require('util');
 const AWS = require('aws-sdk');
 
-const stepfunction = new AWS.StepFunctions();
 const Pooler = require('./pooler.js');
 const replaceError = require('./replace-error.js');
 
 /**
 * @class Worker
 * @param {object} options
 * @param {string} options.activityArn
+* @param {string} [options.region='us-west-2']
```

Comment:
i suggest we do `[options.awsConfig={}]`

It will be better for the following reasons : 
* add both region but also ability to set other aws parameters like 
   - accessKeyId: 'AKID',
   - secretAccessKey: 'SECRET',
* settings region by default to us-west-2 will break the default behavior on servers which is to use the environment variable `AWS_REGION` defined by AWS. SO we should not put any default.